.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Recog Verify",
+            "command": "bin/recog_verify",
+            "type": "process",
+            "isBackground": true,
+            "args": [
+                "--watch",
+                "full",
```

Comment:
VSCode does not support incremental updates to files. Each time any file is modified/added, the entire set of files has to be re-assessed. If only the changed/added file is assessed, VSCode will wipe all problems for other files. Consuming incremental warnings/errors is functionality they're working on providing at some point in the future. I let recog_verify make this configurable so that sometime in the future we can utilize the incremental update mode VSCode may provide.